include/zephyr/bluetooth/iso.h

```diff
@@ -222,7 +228,7 @@ struct bt_iso_recv_info {
 	uint32_t ts;
 
 	/** ISO packet sequence number of the first fragment in the SDU */
-	uint16_t sn;
+	uint16_t seq_num;
```

Comment:
isn't this an API change?

Reply:
It is, but the ISO API is still considered experimental, so we can do changes like this without worrying too much :)

Reply:
ok, I've only checked CONFIG_BT_ISO_UNICAST and BT_ISO_BROADCAST and those aren't market as experimental :)